shadow-dom/reference-target/tentative/label-descendant.html

```diff
@@ -24,15 +24,32 @@
   </div>
 </label>
 
+<label>
+  Input 1 via Options
+  <div>
+    <x-input1 id="x-input1-a"></x-input1>
+  </div>
+</label>
+
+<script>
+  const customInput1 = document.querySelector('x-input1');
```

Comment:
```suggestion
  const customInput1 = document.querySelector('#x-input1-a');
```

As written this will grab the first `<x-input1>`, from the prior test case.